src/silverlabnwb/nwb_file.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, nwb_path, mode='r', verbose=True):
         self.nwb_open_mode = mode
         if mode in {'r', 'r+'} or (mode == 'a' and os.path.isfile(nwb_path)):
             self.open_nwb_file()
+        # assume silverlab extension is in this file's directory
+        load_namespaces(os.path.dirname(os.path.realpath(__file__))+"\\silverlab.namespace.yaml")
```

Comment:
I'd use `os.path.join` to avoid hardcoding to Windows or Linux style paths.

Reply:
Actually, we should be using `pkg_resources` for this so it can still be loaded when the package is installed.

Reply:
See e.g. how `metadata.yaml` is handled in `metadata.py`